src/main/server.ts

```diff
@@ -9,16 +9,16 @@ const port: number = process.env.PORT || 4000
 
 if (app.locals.ENV === 'development') {
   const sslDirectory = path.join(__dirname, 'resources', 'localhost-ssl')
-  const sslOptions = {
+  const serverOptions = {
     key: fs.readFileSync(path.join(sslDirectory, 'localhost.key')),
     cert: fs.readFileSync(path.join(sslDirectory, 'localhost.crt'))
   }
-  const server = https.createServer(sslOptions, app)
+  const server = https.createServer(serverOptions, app)
   server.listen(port, () => {
-    console.log(`Application started: https://localhost:${port}`)
+    console.log(`Application started: https://localhost:${port}/legal`)
```

Comment:
will prefer to have a variable extracted for `legal` and use that in both places.